Algorithm.Python/BasicTemplateFuturesHistoryAlgorithm.py

```diff
@@ -46,32 +46,40 @@ def Initialize(self):
 
         futureGC = self.AddFuture(Futures.Metals.Gold, Resolution.Minute)
         futureGC.SetFilter(timedelta(0), timedelta(182))
-        
+
         self.SetBenchmark(lambda x: 1000000)
 
+        self.Schedule.On(self.DateRules.EveryDay(), self.TimeRules.Every(timedelta(hours=1)), self.MakeHistoryCall)
+        self.successCount = 0
+
+    def MakeHistoryCall(self):
+        history = self.History(self.Securities.keys(), 10, Resolution.Minute)
+        if len(history) < 10:
+            raise Exception(f'Empty history at {self.Time}')
+        self.successCount += 1
+
+    def OnEndOfAlgorithm(self):
+        if self.successCount < 49:
```

Comment:
Nice 👍
A nice improvement would be registering with a comment how did this 49 come to be